### ISSUE TITLE
How is IOS integration going? HELP! OMG! 

### ISSUE THREAD

**Issue by adderly** [kafeg/adctl#6]
Hello @kafeg or @yevgeniy-logachev  or @gilmanov-ildar ,

I was wondering how is the IOS integration going. I will need it and also would help with the integration. If there is some advance into this, please upload it to a "develop" branch so i can finish the integration using the existing code if there is any. 

I would start this week on this, hopefully i get some response on time.

**Comment by kafeg** [comment]
HI, @adderly !

At now we did not has developer who doing iOS integration. Because: I and Ildar didn't has mac os, and other developers didn' has time for porting.

If you want to write port - you may start with code in current master branch. Please fork our repository, start new branch (e.g. iosport), then make first part of porting and give us a pull request. If you have a questions - write me in skype (kafik-fafik).

PS: Sorry for my English =)

**Comment by adderly** [comment]
Yeah, i'll do that. I wont be able to port everything but maybe it serves as a start for others. I will complete my needs right now. Well hopefully you get a vm machine with macosx so you could do some testing soon.

**Comment by FajraKatviro** [comment]
Hi, @adderly ! I've already done preparation part: incapsulated platform specific (obj-C) methods in single class. Just need merge & push & pull request (this weekend)
If you did some work, do we need sync?

**Comment by adderly** [comment]
@FajraKatviro I haven't started. Been busy with my app on android. Great to know that. Hopefully i'll be working on it really soon. I'll wait for you to submit the changes you mentioned, i'll work on that then.

**Comment by kafeg** [comment]
Hi, @adderly! @FajraKatviro give me pull request with platform specific classes and i will merge it into master branch.

You may start making port of library to iOS.

Note: Alredy on iOS work StartAD, AdMob and Google Analytics.

If you have questions - contact with me by skype or by email.

Good luck!

**Comment by adderly** [comment]
@kafeg Okay! Well i guess i what is needed is the google's game related part. Achievement and leaderboards.

**Comment by kafeg** [comment]
@adderly i recommend start with it https://developers.google.com/games/services/ios/quickstart

Also, i think it best to use native SDK versus specialized C++ GPGS SDK, because native SDK use all target platform features.

**Comment by adderly** [comment]
@kafeg Well i did the integration. The android game services libraries i used are the ones provided with the sdk, so many things need to be changed. Also i don't use ant for build nor will test ant tools.

Nevertheless it was a little pain in the arsh with the ios, yet is done. Besides the faulty code which i gracefully deleted, something related to StartAd which is 0% complete so i took the liberty of deleting it.
On monday i'll be editing wrapping up changes, so expect a pull request for this week. 

Funny it took me more than a month to get this done.  :dancers:
